packages/react-devtools-shared/src/devtools/views/Components/SelectedElement.css

```diff
@@ -107,12 +106,26 @@
 }
 
 .OwnerButton {
+  cursor: pointer;
+  width: 100%;
+  padding: 0;
+}
+
+.OwnerContent {
   display: flex;
   align-items: center;
-  margin-left: 0.5rem;
-  padding: 0;
+  padding-top: 0;
+  padding-right: 0;
+  padding-bottom: 0;
```

Comment:
nit: I don't think these three `padding` assignments are required, since `HTMLSpanElement`s don't have any default padding.

Reply:
Woopsie. Those are some leftovers from when I initially started developing the feature. Thanks for making the changes.